test/index.html

```diff
@@ -26,19 +26,20 @@
 	<script src="jquery.js"></script>
 
 	<script>
-		// Load tests if they have not been loaded
-		// This is in a different script tag to ensure that
-		// jQuery is on the page when the testrunner executes
-		// QUnit.config is populated from QUnit.urlParams but only at the beginning
-		// of the test run. We need to read both.
-		var esmodules = QUnit.config.esmodules || QUnit.urlParams.esmodules;
+		( function () {
+			// Load tests if they have not been loaded
+			// This is in a different script tag to ensure that
+			// jQuery is on the page when the testrunner executes
+			// QUnit.config is populated from QUnit.urlParams but only at the beginning
```

Comment:
Can you move the word `beginning` to the next line? It got a bit long (GitHub UI breaks it to a new line for me) and we have space in the next line.